package.json

```diff
@@ -6,6 +6,7 @@
     "@apollo/client": "^3.6.0",
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@material-ui/core": "^4.12.4",
```

Comment:
Seems like a previous version of material was installed by error here

Reply:
you are right, i will update  to the last